packages/api-reference/src/components/Content/Schema/Schema.vue

```diff
@@ -314,6 +340,7 @@ button.schema-card-title:hover {
 }
 .schema-card-description {
   color: var(--scalar-color-2);
+  margin-top: 8px;
```

Comment:
Yo so this is probably adding margin places we don't want it

<img width="3078" height="1968" alt="Image" src="https://github.com/user-attachments/assets/dfee13d9-7876-4d76-ba98-99139a55121d" />

Reply:
whats the fix!

Reply:
I'll push it up